madmom/models

```diff
@@ -1 +1 @@
-Subproject commit c12fab69ae2ebc33965587867f5b230db60a1f08
```

Comment:
You should rebase the whole commit on top of the current master (with updated submodules!) to get rid of this change.